project/ISSUES.md

```diff
@@ -0,0 +1,179 @@
+Galaxy Issue Management
+=======================
+
+The purpose of this document is to formalize how we manage the full cycle
+(reporting, tracking progress, resolution) of both feature/enhancement ideas
+and bugs, as well as providing a few general guidelines for longer term
+planning for various Galaxy related projects.  Significant inspiration taken
+from the way the [Docker](https://github.com/docker/docker) project manages
+issues.
+
+We've tried several different approaches over the course of the project and two
+common problems we've tried to address in this approach are:
+
+* The issue graveyard -- where, once off the first 'page' of issues or below
+  the 'fold' in Trello, things sometimes do not resurface and are lost to
+  history.
+
+* Difficulty clearly presenting and maintaining a high level project roadmap
+  and associated meta-issues.
+
+
+Milestones
+==========
+
+Every pull request should, prior to merge, be assigned to the milestone
+corresponding to the Galaxy release it will first appear in (15.10, 16.01, and
+so on).  This, along with the tags applied, will be used to generate the
+release notes.
+
+Any non-PR issue assigned to a milestone *must* be resolved or reassigned prior
+to publishing that release -- that is, this is the new way to create and track
+"Issues Blocking Release".  This will be a primary way to report bugs and force
```

Comment:
I assume that once merged this document becomes active. Thus I would use 'This is the primary way to force bugs reconciliation' rather than 'This will be.' 
Also this is not a "way to report bugs".